chalicelib/services/JobPostingService.py

```diff
@@ -122,10 +122,14 @@ def getFinanceJobs(self) -> List[Dict]:
         """
         JOBS_SHEET_ID = "15za1luZR08YmmBIFOAk6-GJB3T22StEuiZgFFuJeKW0"
         try:
+            # First get the sheet data with FORMULA render option to parse hyperlink formulas
             response = self.gs.get_all_cells(JOBS_SHEET_ID, "2027 Opportunities Tracker", "FORMULA")
             if not response.get("values"):
                 return []
             
+            # Get the same data with cell hyperlinks (for UI-inserted links)
+            sheet_data = self.gs.get_sheet_with_grid_data(JOBS_SHEET_ID, "2027 Opportunities Tracker")
+            
             # 2. Separate headers from rows
             headers = response["values"][5]
```

Comment:
Unrelated to PR changes, but I'm wondering if we should create consts like HEADER_DIVIDER (maybe there's a better naming) to use instead of numbers so it'd be `response["values"][HEADER_DIVIDER]`. Thoughts?